src/main/java/kea/alog/project/domain/topic/service/TopicService.java

```diff
@@ -10,4 +12,7 @@ PageDto<TopicDto> findAll(Long projectPk, String keyword, TopicSortType sortType
         int size);
 
     TopicDto findOne(Long projectPk, Long topicPk);
+
+    CreateTopicResponseDto create(Long projectPk,
+        CreateTopicRequestDto createTopicRequestDto);
```

Comment:
How about just be projectPk into createTopicRequestDto?

Reply:
I put it in param to use consistent url(/api/projects/{projectPk}/topics)

Reply:
Right. url convention be supposed to keep well.